src/components/molecules/Visualizer/Infobox/index.tsx

```diff
@@ -65,6 +64,7 @@ const Infobox: React.FC<Props> = ({
   onBlockMove,
   renderInsertionPopUp,
   onBlockInsert,
+  ...props
```

Comment:
Can you rename this to `pluginProps`, `pluginCommonProps` or `commonProps` ? Since I think all the remaining props belong to PluginCommonProps, it would be clearer what they are.
(I'm not QUITE sure this is necessary, so if you feel like this is a bad idea or feel like it isn't quite right please let me know)

Reply:
I think rename is better for understanding if the `...props` only refers to `pluginCommonProps`. I'll check & update.

Reply:
No, I think you are right to leave it. If the types was never going to be added to, or we could control better what ...props contains, perhaps what I suggested is good. But since we can't and if I add an additional prop to the Props type it would then be a part of the spread props, I think its better to leave it as is. Thanks for your feedback!